### PR TITLE
Loading translations in provider, added namespace prefix aswell.

### PR DESCRIPTION
**Pull request by gluttons**
Loading translations from provider.
Updated install command for 5.4

### CODE REVIEW

File: src/Commands/InstallCommand.php [hunk 1]
```diff
@@ -69,7 +70,19 @@ public function handle(Filesystem $filesystem)
         $this->call('hook:setup');
 
         $this->info('Publishing the Voyager assets, database, language, and config files');
-        $this->call('vendor:publish', ['--provider' => VoyagerServiceProvider::class]);
+
+        //Publish only relevant resources on install
+        $tags = ['voyager_assets', 'seeds', 'demo_content', 'config', 'migrations', 'lang'];
+
+        if (app()->version() >= 5.4) {
+            $tags = array_diff($tags, ['lang']);
```
Comment (fletch3555) [inline src/Commands/InstallCommand.php]: Same comments as here: https://github.com/the-control-group/voyager/pull/2343#pullrequestreview-82862437.  Also, having this here as well will cause merge conflicts for whichever PR gets merged last (this or #2343)

File: src/VoyagerServiceProvider.php [hunk 2]
```diff
@@ -248,7 +249,7 @@ private function registerPublishableResources()
                 "{$publishablePath}/config/voyager.php" => config_path('voyager.php'),
             ],
             'lang' => [
-                "{$publishablePath}/lang/" => base_path('resources/lang/'),
+                "{$publishablePath}/lang/" => resource_path('lang/vendor/voyager'),
```
Comment (fletch3555) [inline src/VoyagerServiceProvider.php]: I like the idea of keeping translations inside the package and namespacing them, however, then we should not allow publishing them.  Much like views, devs can simply copy the file to `resources/lang/vendor/voyager/` on an as-needed basis.  Publishing them all will provide a permanent override and will make future upgrades difficult.  I suggest removing the `lang` entry altogether

**Review by fletch3555 — CHANGES_REQUESTED** [review]
This is a LOT of changes, so I haven't had the chance to look through them all.  Most are just changing the lang references, so that should be fine.

**Comment by gluttons** [comment]
I've updated the PR with the suggested changes!

**Review by fletch3555 — DISMISSED** [review]

**Comment by fletch3555** [comment]
Bumping this to the v1.1 stream so users expect some backwards incompatibility.  There are some merge conflicts due to this change.  @gluttons, Please resolve them.  Thank you

**Comment by gluttons** [comment]
I've been swamped at work the last couple of days, will come back and fix this PR as soon as I can.

**Comment by gluttons** [comment]
You've accidentally accepted a PR with a merge-diff still present into the 1.x-dev branch, i fixed the issue in this PR. (else styleci would be so angry at me)

**Comment by fletch3555** [comment]
You have some merge conflicts now that #2343 has been merged.  Please resolve these and we'll get this merged in as well.

**Comment by fletch3555** [comment]
@gluttons, I did the merge for you, but it looks like [tests are failing](https://travis-ci.org/the-control-group/voyager/jobs/330091773) since it can't find a "Save" button on certain pages.  Can you take a look? Sounds like it's related to your views, but it's possible I broke something when dealing with the merge conflict

**Comment by fletch3555** [comment]
See if you can figure it out, but the test didn't fail for me until I ran composer update, so it may be due to a change in some dependency.  For the record, here's what composer updated:

```
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 0 installs, 32 updates, 0 removals
  - Updating symfony/css-selector (v3.4.2 => v3.4.3): Downloading (100%)
  - Updating tijsverkoyen/css-to-inline-styles (2.2.0 => 2.2.1): Downloading (100%)
  - Updating symfony/var-dumper (v3.4.2 => v3.4.3): Downloading (100%)
  - Updating symfony/routing (v3.4.2 => v3.4.3): Downloading (100%)
  - Updating symfony/process (v3.4.2 => v3.4.3): Downloading (100%)
  - Updating symfony/http-foundation (v3.4.2 => v3.4.3): Downloading (100%)
  - Updating symfony/event-dispatcher (v4.0.2 => v4.0.3): Downloading (100%)
  - Updating symfony/debug (v3.4.2 => v3.4.3): Downloading (100%)
  - Updating symfony/http-kernel (v3.4.2 => v3.4.3): Downloading (100%)
  - Updating symfony/finder (v3.4.2 => v3.4.3): Downloading (100%)
  - Updating symfony/console (v3.4.2 => v3.4.3): Downloading (100%)
  - Updating ramsey/uuid (3.7.1 => 3.7.2): Downloading (100%)
  - Updating symfony/translation (v3.4.2 => v3.4.3): Downloading (100%)
  - Updating doctrine/inflector (v1.2.0 => v1.3.0): Downloading (100%)
  - Updating laravel/framework (v5.5.25 => v5.5.31): Downloading (100%)
  - Updating seld/jsonlint (1.6.2 => 1.7.0): Downloading (100%)
  - Updating composer/spdx-licenses (1.1.6 => 1.2.0): Downloading (100%)
  - Updating symfony/filesystem (v3.4.2 => v4.0.3): Downloading (100%)
  - Updating composer/composer (1.5.5 => 1.6.2): Downloading (100%)
  - Updating larapack/hooks (v1.0.1 => v1.0.4): Downloading (100%)
  - Updating larapack/voyager-hooks (v1.0.4 => v1.0.6): Downloading (100%)
  - Updating sebastian/comparator (2.1.0 => 2.1.2): Downloading (100%)
  - Updating phpunit/phpunit-mock-objects (5.0.5 => 5.0.6): Downloading (100%)
  - Updating phpunit/phpunit (6.5.4 => 6.5.5): Downloading (100%)
  - Updating orchestra/testbench-core (v3.5.4 => v3.5.6): Downloading (100%)
  - Updating orchestra/testbench (v3.5.2 => v3.5.3): Downloading (100%)
  - Updating symfony/dom-crawler (v3.4.2 => v3.4.3): Downloading (100%)
  - Updating orchestra/testbench-browser-kit (v3.5.3 => v3.5.4): Downloading (100%)
  - Updating symfony/config (v4.0.2 => v4.0.3): Downloading (100%)
  - Updating symfony/stopwatch (v4.0.2 => v4.0.3): Downloading (100%)
  - Updating symfony/yaml (v4.0.2 => v4.0.3): Downloading (100%)
  - Updating phpunit/phpcov dev-master (522e81f => 1fc7ed3):  Checking out 1fc7ed3179
Package guzzle/guzzle is abandoned, you should avoid using it. Use guzzlehttp/guzzle instead.
Writing lock file
Generating autoload files
```

**Review by fletch3555 — APPROVED** [review]

**Pull request merged**